HW2.js

```diff
@@ -0,0 +1,68 @@
+// Task #1 - Задача про обчислення різниці часу
+
+    function durationBetweenDates ( dateStart = '05 Oct 2020', dateEnd = '06 Oct 2023', dimension = 'days'){
+
+        let timeDiff =  Math.abs(new Date(dateEnd) - new Date(dateStart)); 
+
+        switch (dimension){
+            case ('days'): timeDiff /= (1000 * 3600 * 24); break;
```

Comment:
Division assignment - перший раз бачу в домашках, прикольно)

В решті краще форматуйте код, з переносами, так буде майже всюди ок
```suggestion
        case 'days':
            timeDiff /= (1000 * 3600 * 24);
            break;
```